src/views/record-browser/record-browser.scss

```diff
@@ -62,4 +62,57 @@ $sidebar-width: 300px;
 			&.sep { color: $gray-medium; }
 		}
 	}
+
+	thead {
```

Comment:
All of these table styles are specific to the `table-data` component. Move them to `src/views/record-browser/table-data/table-data.scss`